.github/workflows/merge.yml

```diff
@@ -0,0 +1,13 @@
+name: Merge
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  auto_approve:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'reinaris' }}
```

Comment:
I noticed, while working on some other projects, that these workflows just run even if you propose a PR 🤔 But glad to see that this doesn't work on projects you don't own

Reply:
Ah I see more creative workflow PR's now https://github.com/Hacker0x01/hacker101/pull/208 :)

Reply:
That's interesting. Curious about what happens if I do allow the workflows to run. Workflows don't run by default when they come from an external contributor. I guess the other reason for that is to avoid huge costs if someone floods you with a lot of PRs

Reply:
Anyways, I succeeded in saying Hi to Willie ;)